Winium/Winium.StoreApps.Driver/EmulatorHelpers/Deployer.cs

```diff
@@ -155,6 +155,25 @@ public void SendFiles(Dictionary<string, string> files)
             }
         }
 
+        public void InstallDependencies(List<string> dependencies )
+        {
+            if (dependencies == null || !dependencies.Any())
+            {
+                return;
+            }
+
+            foreach (var dependency in dependencies)
+            {
+                InstallDependency(dependency);
+            }
+        }
+
+        public void InstallDependency(string path)
+        {
+            var appManifest = Utils.ReadAppManifestInfoFromPackage(path);
+            Utils.InstallApplication(this.deviceInfo, appManifest, DeploymentOptions.None, path);
+        }
```

Comment:
I think we can reuse this method for `Install` too, so that we have private `InstallApplicationPackage(string path)` method and public `Install(string app)` and `InstallDependencies(List<string> dependencies )` methods, that reuse `InstallApplicationPackage `.